src/Raven.Server/Documents/Indexes/Static/StaticIndexBase.cs

```diff
@@ -318,7 +318,7 @@ protected dynamic TryConvert<T>(object value)
                     return l;
                 if (v is float f)
                     return f;
-                if (v is LazyStringValue lsv && double.TryParse(lsv, out var r))
+                if (v is LazyStringValue lsv && double.TryParse(lsv.AsReadOnlySpan(), out var r))
                     return r;
```

Comment:
Shouldn't this be applied independently?

Reply:
Yes, but it won't compile otherwise :-)
Was merged seprately, IIRC